### DISCUSSION TITLE
Changing the color of the burned particles

### DISCUSSION THREAD

**Discussion by Sadeghi6872** [firemodels/fds#13144] [General]
Dear all,
![Color](https://github.com/firemodels/fds/assets/175089980/ce3df74d-e391-4660-9022-dba52ca4e880)
 

I hope you are doing well. I have simulations done in FDS on the excelsior particles. The primary color of the particles is green as shown in the screenshot I have attached. How can I change the burned particles to show that they are burned! I would appreciate any advice. 

Best regards,
Mohamad

**Comment by drjfloyd** [comment]
You could pick a particle output quantity like diameter, mass, or bulk density and figure out what value represents a burned particle. Then run FDS, load the particle file, and show that quantity. Smokeview will default to using a rainbow color bar, but you can create your own color bar. Details are in the Smokeview User's Guide.
